File: solutions/2026-05-30/Q10.py
# Problem: Print prime numbers in a range.

a, b = map(int, input().split())
print(
    *(
        i
        for i in range(a, b + 1)
        if i > 1 and all(i % j != 0 for j in range(2, int(i**0.5) + 1))
    )
)

File: solutions/2026-05-30/Q10.c
/*
 * Problem: Print prime numbers in a range.
 */

#include <stdio.h>
int main() {
  int a, b;
  scanf("%d%d", &a, &b);
  for (int i = a; i <= b; i++) {
    int f = 1;
    for (int j = 2; j * j <= i; j++)
      if (i % j == 0)
        f = 0;
    if (f && i > 1)
      printf("%d ", i);
  }
  return 0;
}
